src/locales/fr.json

```diff
@@ -369,7 +369,9 @@
       "awaiting": "Catégorisation en cours"
     },
     "virtualAccounts": {
-      "healthReimbursements": "Remb santé en attente"
+      "healthReimbursements": "Remb santé en attente",
```

Comment:
Are we sure we do not want "Remb. santé en attente" (with a dot) ?

Reply:
I will see with Flo